clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesNamedAccountCredentials.java

```diff
@@ -63,6 +63,14 @@ public KubernetesNamedAccountCredentials(AccountCredentialsRepository accountCre
     // TODO(lwander): what is this?
     this.requiredGroupMembership = requiredGroupMembership == null ? Collections.emptyList() : Collections.unmodifiableList(requiredGroupMembership);
     this.dockerRegistries = dockerRegistries != null ? dockerRegistries : new ArrayList<>();
+
+    for (int i = 0; i < this.dockerRegistries.size(); i++) {
+      LinkedDockerRegistryConfiguration registry = this.dockerRegistries.get(i);
```

Comment:
Ah sorry, I should have looked into that - I assumed it was the groovy foreach which doesn't let you modify elements. I'll fix this up in a cleanup PR.